dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-voice/src/main/java/org/apache/dolphinscheduler/plugin/alert/voice/VoiceAlertChannel.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.alert.voice;
+
+import static org.apache.dolphinscheduler.plugin.alert.voice.VoiceAlertConstants.NAME_ACCESS_KEY_ID;
+import static org.apache.dolphinscheduler.plugin.alert.voice.VoiceAlertConstants.NAME_ACCESS_KEY_SECRET;
+import static org.apache.dolphinscheduler.plugin.alert.voice.VoiceAlertConstants.NAME_ADDRESS;
+import static org.apache.dolphinscheduler.plugin.alert.voice.VoiceAlertConstants.NAME_CALLED_NUMBER;
+import static org.apache.dolphinscheduler.plugin.alert.voice.VoiceAlertConstants.NAME_CALLED_SHOW_NUMBER;
+import static org.apache.dolphinscheduler.plugin.alert.voice.VoiceAlertConstants.NAME_TTS_CODE;
+
+import org.apache.dolphinscheduler.alert.api.AlertChannel;
+import org.apache.dolphinscheduler.alert.api.AlertInfo;
+import org.apache.dolphinscheduler.alert.api.AlertResult;
+
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class VoiceAlertChannel implements AlertChannel {
+
+    @Override
+    public AlertResult process(AlertInfo info) {
+
+        Map<String, String> paramsMap = info.getAlertParams();
+        if (null == paramsMap) {
+            return new AlertResult("false", "aliyun-voice params is null");
+        }
+        VoiceParam voiceParam = buildVoiceParam(paramsMap);
+        VoiceSender voiceSender = new VoiceSender(voiceParam);
+
+        AlertResult alertResult = voiceSender.send();
+        boolean flag = Boolean.parseBoolean(String.valueOf(alertResult.getStatus()));
+        if (flag) {
+            log.info("alert send success");
+            alertResult.setStatus("true");
+            alertResult.setMessage("vocie send success.");
+        } else {
+            alertResult.setMessage("vocie send error.");
+            log.info("alert send error : {}", alertResult.getMessage());
+        }
```

Comment:
This logic does not very grace, please refer `sendDingTalkMsg` method

Reply:
completed